### DISCUSSION TITLE
Seeing only public repositories in organization profile

### DISCUSSION THREAD

**Discussion by ghost** [community/community#37501] [Other Feature Feedback, Questions, & Ideas]
Hi, I've encountered this issue for the last few days: I belong to an organization and, usually, when I navigate to the home page of the organization (its profile) I usually see all the most recently modified repositories (both public or private). It's a few days that when I navigate to that same page I only see the public repositories, and I have to filter first for Public repositories then to "All" to reset the list behaviour. This is pretty annoying since I usually use that page to navigate quickly to all the organization repos.

**Comment by ghost** [comment]
I've seen that switching to "Member" view I can see all the repositories as before. Maybe the default organization view changed?

**Comment by ossanna16** [comment]
Hi there @giovanni-bertoncelli and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
